docs/docs/accessibility.md

```diff
@@ -0,0 +1,277 @@
+---
+id: accessibility
+title: Accessibility
+permalink: docs/accessibility.html
+redirect_from:
+  - "docs/index.html"
+---
+
+## Why accessibility?
```

Comment:
Nit: we generally use Capitalized Style for large headers. So that would be "Why Accessibility?"